src/getNodeByIdRef.ts

```diff
@@ -5,5 +5,5 @@ export function getNodeByIdRef({ container, idRef }) {
     return null;
   }
 
-  return container.querySelector(`#${idRef}`);
+  return container.querySelector(`#${CSS.escape(idRef)}`);
```

Comment:
```suggestion
  return container.getElementById(idRef);
```

Nothing wrong here per say, but as mentioned, suspect this would be subtly more performant in avoiding the need to escape.